lib/deaf_grandma.rb

```diff
@@ -3,9 +3,11 @@
 #CAREFUL! This script will not exit. Do you know why? You may have to close it with `Ctrl-C` (Mac) if you do not insert an `exit` into your speak method.
 
 class DeafGrandma
+  attr_reader :SpeakResult
 
   def initialize
     @bye_counter = 0 
+    @SpeakResult = Struct.new(:done, :reply)
```

Comment:
This could be pulled up to a constant. That would remove the `@` character, yay!

I don't think this refactor was a clear win. I kinda like it, but not by much. Sometimes I like to try different options and see what feels right. I would be fine with leaving it the other way, too.

Thanks for trying it!